lib/ExpressionUtils/core/exprMatrix_utils.py

```diff
@@ -203,3 +207,117 @@ def get_expression_matrix(self, params):
                                                   expr_set_data,
                                                   '{0}_TPM_ExpressionMatrix'.format(output_obj_name))
         return fpkm_ref, tpm_ref
+
+    # returns a list of [ min, max, mean, std.dev, is_data_missing] for one row of conditional 
```

Comment:
The preferred way to document python functions is with triple double quoted doc strings. See the validate_params function at L33 for an example